inst/qml/RaterAgreement.qml

```diff
@@ -39,7 +39,59 @@ Form
 
 	Section
 	{
-		title: 	qsTr("Intraclass Correlation Coefficients")
+		title: 	qsTr("Intraclass Correlation")
 
+		Group
+		{
+
+			RadioButtonGroup
```

Comment:
Maybe it is worthwhile to think about displaying three ICCs at a time, so users could see the difference when they change their assumptions about the raters. However, this could also mislead users to always change their assumptions to yield the highest ICC. I wonder how SPSS does it? Is the type of ICC predetermined by the research question, or can users actually change that...

Reply:
I think in most cases the ICC is predetermined, which is why I split them up to discourage just searching for the highest one. SPSS seems to show you two ICCs at a time: the ICCX,1 and the ICCX,k which might be a nice middle route as it's not that ambiguous whether ratings are averaged or not. 🤔 Another reason why I quite like splitting the ICCs up is that this allows the UI to include some information about what type of ICC you are running depending on the boxes you checked

Reply:
Good. Let's leave it as is then.